cluster/gce/util.sh

```diff
@@ -500,9 +501,126 @@ function write-node-env {
     KUBERNETES_MASTER_NAME="${MASTER_NAME}"
   fi
 
+  construct-kubelet-flags false
   build-kube-env false "${KUBE_TEMP}/node-kube-env.yaml"
 }
 
+# $1: if 'true', we're rendering flags for a master, else a node
+function construct-kubelet-flags {
+  local master=$1
+  local flags="${KUBELET_TEST_LOG_LEVEL:-"--v=2"} ${KUBELET_TEST_ARGS:-}"
+  flags+=" --allow-privileged=true"
+  flags+=" --cgroup-root=/"
+  flags+=" --cloud-provider=gce"
+  flags+=" --cluster-dns=${DNS_SERVER_IP}"
+  flags+=" --cluster-domain=${DNS_DOMAIN}"
+  flags+=" --pod-manifest-path=/etc/kubernetes/manifests"
+  # Keep in sync with CONTAINERIZED_MOUNTER_HOME in configure-helper.sh
+  flags+=" --experimental-mounter-path=/home/kubernetes/containerized_mounter/mounter"
+  flags+=" --experimental-check-node-capabilities-before-mount=true"
+  # Keep in sync with the mkdir command in configure-helper.sh (until the TODO is resolved)
+  flags+=" --cert-dir=/var/lib/kubelet/pki/"
+
+  # TODO(mtaufen): KUBELET_PORT looks unused; delete it?
```

Comment:
What calls into `test/kubemark/resources/start-kubemark-master.sh`?

Reply:
Looks like the startup script for kubemark on GCE: https://github.com/kubernetes/kubernetes/blob/4a6fec738b16db1f26ca968ab22984aec4e817ca/test/kubemark/gce/util.sh#L71

But I can't see anywhere that the var gets changed there either. Maybe it can be removed from both places?

Reply:
It should be fine to remove. I didn't find any occurrences of `KUBELET_PORT` in test-infra either, so I don't think any test jobs are configuring it.

Reply:
removed.